api/LargestContentfulPaint.json

```diff
@@ -249,7 +249,7 @@
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": false
+                "version_added": "122"
```

Comment:
It looks like Firefox 122 did not support it yet:

<img width="813" height="342" alt="image" src="https://github.com/user-attachments/assets/b4bb0fcc-f73b-4c50-a38b-ca136f848387" />

Based on testing in BrowserStack Live, it appears to have been first supported in Firefox 141, which correlates with [bug 1963917](https://bugzilla.mozilla.org/show_bug.cgi?id=1963917).

```suggestion
                "version_added": "141"
```